packages/muelu/test/interface/default/MLParameterListInterpreter/MLaux.xml

```diff
@@ -4,5 +4,7 @@
   <Parameter name="coarse: max size"            type="int"      value="100"/>
   <Parameter name="repartition: enable"         type="int"      value="1"/>
   <Parameter name="aggregation: aux: enable"    type="bool"     value="true"/>
+  <!-- Set threshold to something weird, since aux: threshold should be used. -->
+  <Parameter name="aggregation: threshold"      type="double"   value="-1e15"/>
```

Comment:
Does the XML translator parse this correctly?

Reply:
With the change to ML2MueLuParameterTranslator, it should see that aux aggregation is enabled, and then overwrite "aggregation: threshold" with "aggregation: aux: threshold" before translating that to "aggregation: drop tol". What happened before is that "sa: use filtered matrix" was always on, and it would use "aggregation: threshold" for the drop tol.